frontend/packages/data-portal/app/components/Layout/Footer.tsx

```diff
@@ -76,17 +76,16 @@ export function Footer() {
   )
 
   return (
-    <footer className="bg-sds-gray-black min-h-[213px] p-sds-xxl flex flex-col flex-shrink-0">
-      <div className="flex items-center flex-wrap flex-col sm:flex-row sm:gap-y-sds-xxl">
+    <footer className="bg-sds-gray-black min-h-[213px] pt-[41px] pb-sds-xxl px-sds-xl screen-716:px-sds-xxl flex flex-col flex-shrink-0">
```

Comment:
can be done later, but if `716` is the breakpoint we're going to settle on for phones it might make sense to add it to our tailwind config

Reply:
Oh I just realized this is the alias 😆. Maybe we should give them better semantics like "phone", "desktop" or whatever, but that's not really relevant to this PR

Reply:
@codemonkey800 SDS has had breakpoints on its backlog for a long time -- im not sure if that work has been completed (Connor was scoping it, last I heard). We'd want to defer to SDS when they have defined them. 

Agreed, mobile has not been generally considered a priority, ML challenge page was a special case because of how the page is promoted.